pkg/server/server_test.go

```diff
@@ -20,7 +21,7 @@ func TestRunServerSuite(t *testing.T) {
 	suite.Run(t, &TestServerSuite{test.UnitTestSuite{}})
 }
 
-func (s *TestServerSuite) TestServer() {
+func (s *TestServerSuite) TestCors() {
```

Comment:
Since the cors test is just a subtest of that bigger server test it would make sense to keep the original test name :)

```suggestion
func (s *TestServerSuite) TestServer() {
```

Reply:
accidentally happened when i merged :)